src/CodeGen.fs

```diff
@@ -1203,4 +1214,73 @@ type {clientName}(url: string, httpClient: HttpClient) =
 let sampleFSharpGraphqlClient projectName clientName errorType members serializer =
     match serializer with
     | SerializerType.System -> sampleFSharpSystemGraphqlClient projectName clientName errorType members
-    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
\ No newline at end of file
+    | SerializerType.Newtonsoft -> sampleFSharpNewtonsoftGraphqlClient projectName clientName errorType members
+
+let sampleFableGraphqlClientFsi projectName clientName =
+        $"""namespace {projectName}
+
+open Newtonsoft.Json
+open Newtonsoft.Json.Linq
+open Fable.Remoting.Json
+open System.Net.Http
+open System.Text
+
+type {clientName} =
+    class
+        /// <summary>Creates {clientName}</summary>
+        /// <remarks>
+        /// In order to enable all F# types serialization and deserealization you must add 
+        /// <see href="T:Fable.Remoting.Json.FableJsonConverter">FableJsonConverter</see>
+        /// from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a> NuGet package
```

Comment:
```suggestion
        /// In order to enable all F# types serialization and deserealization 
        /// <see href="T:Fable.Remoting.Json.FableJsonConverter">FableJsonConverter</see> is added
        /// from <a href="https://github.com/Zaid-Ajaj/Fable.SimpleJson">Fable.SimpleJson</a> NuGet package
```

Reply:
These remarks are not entirely correct, the type `FableJsonConverter` from `Fable.Remoting.Json` is not from `Fable.SimpleJson`. 
 - Fable.Remoting.Json is a F# JSON converter running on dotnet
 - Fable.SimpleJson is F# JSON converter running with Fable

Reply:
Ok, I'll redo